packages/mds-policy-service/service/validators.ts

```diff
@@ -14,33 +14,126 @@
  * limitations under the License.
  */
 
-import { schemaValidator } from '@mds-core/mds-schema-validators'
-import Joi from 'joi'
+import { SchemaValidator } from '@mds-core/mds-schema-validators'
+import {
+  ACCESSIBILITY_OPTIONS,
+  DAYS_OF_WEEK,
+  MICRO_MOBILITY_VEHICLE_EVENTS,
+  MICRO_MOBILITY_VEHICLE_STATES,
+  MODALITIES,
+  RULE_TYPES,
+  TAXI_VEHICLE_EVENTS,
+  TAXI_VEHICLE_STATES,
+  TNC_VEHICLE_EVENT,
+  TNC_VEHICLE_STATE
+} from '@mds-core/mds-types'
 import { PolicyDomainModel, PolicyMetadataDomainModel } from '../@types'
 
+const stringSchema = (options = {}) => ({ type: 'string', ...options })
+const uuidSchema = { type: 'string', format: 'uuid' }
+// Timestamp Schema that ensures milliseconds
+const TimestampSchema = (options = {}) => ({
+  type: 'integer',
+  minimum: 100_000_000_000,
+  maximum: 99_999_999_999_999,
+  ...options
+})
+
+const arraySchema = (items = {}, options = {}) => ({
+  type: 'array',
+  items,
+  ...options
+})
+
+const enumSchema = <T>(enumType: T[]) => ({ type: 'string', enum: enumType })
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+const micromobilityStateMap = MICRO_MOBILITY_VEHICLE_STATES.reduce<any>((acc, state) => {
```

Comment:
Would be nice to make a better type for this, something like `{ [k in MICROMOBILITY_VEHICLE_STATES}: JsonSchemaObject }` (or whatever the AJV schema type is)